lib/new-admin/graphql/resolvers/bill.resolver.js

```diff
@@ -2,7 +2,9 @@ const bills = require('../../services/bills')
 
 const resolvers = {
   Query: {
-    bills: () => bills.getBills()
+    bills: () => bills.getBills(),
+    looseBills: () => bills.getLooseBills(),
+    looseBillsByMachine: (...[, { deviceId }]) => bills.getLooseBillsByMachine(deviceId)
```

Comment:
My feeling is that this should be the same graphql endpoint with different filters.